src/unitTest/PageParserTest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+include_once '../parserPages/PageParser.php';
+include_once '../dataBase/XmlFeedModel.php';
+
+use PHPUnit\Framework\TestCase;
+
+class PageParserTest extends TestCase
+{
+    public function testPageParser()
+    {
+        $objectPage = new PageParser();
+        $sourceLink = 'https://usn.ubuntu.com/usn/rss.xml';
+        $this->assertNotEmpty($sourceLink, 'This link empty!');
```

Comment:
What is the purpose of this assert? Which part of your library's code are you testing ?